### ISSUE TITLE
Unable to update translations after creations

### ISSUE THREAD

**Issue by chrkaatz** [directus/api#1970]

**Comment by chrkaatz** [comment]
@rijkvanzanten any Idea what could be the issue

**Comment by rijkvanzanten** [comment]
No.. Somewhere in the API it's converting the nested JSON structure into SQL queries wrong. I'll check to make sure this doesn't occur in [v9](https://github.com/directus/next).

**Comment by chrkaatz** [comment]
@rijkvanzanten How stable is v9 at this point?

**Comment by rijkvanzanten** [comment]
The features that have been implemented are fairly stable at this point. 
There's still a couple pieces missing though, so depending on what features you rely on, it might not be ready for "real" use yet

**Comment by jojomillenium** [comment]
Same issue here with same directus version (8.8)
I implemented the translation feature as per the documentation.
When I create a new entry in my "Article" collection, I fill in the name, and the content in the translated field (as per @chrkaatz screenshot) => save successful on new entry creation.
But If I open my entry and want to edit a translated text => "Failed generating the SQL entry".

Any progress on this? 
I saw @rijkvanzanten comments on v9, seems to also happen there.

**Comment by jojomillenium** [comment]
FYI, there is a work around to delete the field "created_on" from the parent and child table.
